### DISCUSSION TITLE
Status Alarm

### DISCUSSION THREAD

**Discussion by anelson61** [arkypita/LaserGRBL#994] [Q&A]
Hello I'm a newbie to laser engraving,  I purchased a Stump Laser, I thought I got it setup correctly,  I'm able to connect to the laser, however when I do the status says "Alarm"

**Comment by gmmanonymus111** [comment]
This should be the intened function of GRBL. Once powered on, it goes to an alarm status, and needs to be unlocked. You can click on the `Lock` icon to unlock it. If you have limit switches, and homing enabled, you can click on the `House` icon, this also unlocks your machine.(by `House` icon, I mean the one at custom buttons, not at jog control)
